crypto/keyring/keyring.go

```diff
@@ -581,3 +582,7 @@ func newRealPrompt(dir string, buf io.Reader) func(string) (string, error) {
 		}
 	}
 }
+
+func addrKey(address types.AccAddress) []byte {
+	return []byte(fmt.Sprintf("%s.%s", hex.EncodeToString(address.Bytes()), addressSuffix))
```

Comment:
why encode to hex when we can just use the `String` interpretation?

Reply:
Because `String` depends on the notorious `Config` singleton

Reply:
I didn't change it for the old Keybase implementations to avoid breaking `gaia` versions used for Cosmos Hub